BaragonUI/app/actions/api/services.es6

```diff
@@ -29,3 +31,20 @@ export const ReloadService = buildJsonApiAction(
     url: `/state/${serviceId}/reload`,
   })
 );
+
+export const RemoveUpstreams = buildJsonApiAction(
+  'REMOVE_UPSTREAMS',
+  'POST',
+  (loadBalancerService, upstreams, noValidate, noReload) => ({
+    url: '/request',
+    body: {
+      loadBalancerService,
+      noValidate,
+      noReload,
+      loadBalancerRequestId: buildRequestId(loadBalancerService.serviceId),
+      addUpstreams: [],
+      removeUpstreams: upstreams,
+    },
+  }),
+  (serviceId) => serviceId,
```

Comment:
It would make more sense to have this keyed on the requestId. When fetching results for the request we are going to have to look it up by that id, not the serviceId